spec.md

```diff
@@ -568,15 +499,13 @@ Such actions MAY include, but SHALL NOT be limited to, the following:
 * Notifying the plugin supervisor.
 
 The Plugin MAY verify that it has the right configurations, devices, dependencies and drivers in order to run and return a success if the validation succeeds.
-The CO MAY invoke this RPC at any time after version negotiation has been completed (see `GetSupportedVersions`).
+The CO MAY invoke this RPC at any time.
```

Comment:
If we decide to add a `Versioned` service later (and add back `GetSupportedVersions`) there will be ordering dependencies here.

Reply:
I think that's ok...we would just need to add the version with a different position id.  In  other words, the following is totally valid:
```
message ControllerUnpublishVolumeRequest {
  Version version = 4;
  string volume_id = 1;
  ...
}
```